go.mod

```diff
@@ -126,7 +126,7 @@ require (
 	github.com/DataDog/datadog-agent/pkg/util/scrubber v0.58.0 // indirect
 	github.com/DataDog/go-sqllexer v0.0.14 // indirect
 	github.com/DataDog/go-tuf v1.1.0-0.5.2 // indirect
-	github.com/DataDog/opentelemetry-mapping-go/pkg/otlp/attributes v0.20.0 // indirect
+	github.com/Masterminds/semver/v3 v3.3.1 // indirect
```

Comment:
This should be a one-time change on the initial PR (since the workflow uses this package)